packages/eas-cli/src/credentials/ios/appstore/provisioningProfileAdhoc.ts

```diff
@@ -270,10 +246,6 @@ export async function createOrReuseAdhocProvisioningProfileAsync(
       spinner.succeed(`Used existing profile: ${profileName}`);
     }
 
-    delete adhocProvisioningProfile.didUpdate;
-    delete adhocProvisioningProfile.didCreate;
-    delete adhocProvisioningProfile.profileName;
```

Comment:
I think we need to leave this. Those properties are still returned from `manageAdHocProfilesAsync`.

Reply:
See https://github.com/expo/eas-cli/blob/9d8e64da17814338b61885b42647b9a2631c9bc1/packages/eas-cli/src/credentials/ios/appstore/provisioningProfileAdhoc.ts#L230-L235

I now pull them out by destructuring instead of mutating the original object.